### PR TITLE
Updated couchdb 3.3.2, 3.3, 3, latest, 3.2.3 and 3.2

### PR DESCRIPTION
**Pull request by nickva**

### CODE REVIEW

File: library/couchdb [hunk 1]
```diff
@@ -4,15 +4,15 @@
 Maintainers: Joan Touzet <wohali@apache.org> (@wohali), Jan Lehnardt <jan@apache.org> (@janl), Nick Vatamaniuc (@nickva)
 GitRepo: https://github.com/apache/couchdb-docker
 GitFetch: refs/heads/main
-GitCommit: a29d12f0dd4d73f1dcefb07212af4b9ba3185d9e
+GitCommit: 72a0aebfa3248b3df64a70049f0fb1f90c042a49
 
 Tags: latest, 3.3.1, 3.3, 3
```
Comment (yosifkit) [inline library/couchdb:9]:
```diff
-Tags: latest, 3.3.1, 3.3, 3
+Tags: latest, 3.3.2, 3.3, 3
```
This needs a tag bump too.

Comment (nickva) [reply library/couchdb]: Oh my bad. Thanks for noticing, @yosifkit. Fixed it!

File: library/couchdb [hunk 2]
```diff
@@ -4,15 +4,15 @@
...
 
-Tags: latest, 3.3.1, 3.3, 3
+Tags: latest, 3.3.2, 3.3, 3
 Architectures: amd64, arm64v8, ppc64le
-Directory: 3.3.1
+Directory: 3.3.2
 
-Tags: 3.2.2, 3.2
+Tags: 3.2.3, 3.2
 Architectures: amd64, arm64v8, ppc64le
```
[the remark this answers is not in the document]
Comment (nickva) [reply library/couchdb:14]: There is no upstream infra, it's all manual via a script :-). It was just a mistake. The latest 3.3.3 version has 4 architectures: https://hub.docker.com/layers/apache/couchdb/3.3.3/images/sha256-982d410b5e7eb76eb55128a4555bf8e44be5c02620f039d31da85555669432ee?context=repo

[the remark this answers is not in the document]
Comment (tianon) [reply library/couchdb:14]: Ah fair :smile:

3.3.3 does look fine, but 3.2.3 is missing everything other than amd64 :sweat_smile:

**Pull request merged**